testdata/resources/docker_service/updateFailsAndRollbackConvergeConfig.tf

```diff
@@ -5,12 +5,25 @@ provider "docker" {
   }
 }
 
+data "docker_registry_image" "tftest_image" {
+  provider             = "docker.private"
+  name                 = "%s"
+  insecure_skip_verify = true
+}
+resource "docker_image" "tftest_image" {
+  provider      = "docker.private"
+  name          = data.docker_registry_image.tftest_image.name
+  keep_locally  = false
+  force_remove  = true
+  pull_triggers = [data.docker_registry_image.tftest_image.sha256_digest]
+}
+
 resource "docker_service" "foo" {
   provider = "docker.private"
   name     = "tftest-service-updateFailsAndRollbackConverge"
   task_spec {
     container_spec {
-      image             = "%s"
+      image             = docker_image.tftest_image.latest
```

Comment:
```suggestion
      image             = docker_image.tftest_image.id
```

Reply:
see 58a5e8599beaabff8aedab2682e1e894dbaddc93 and 3a6790cf1eb69279e2a0b37ad49d8a2d89f2c5f2